org.adempiere.base/src/org/compiere/util/Util.java

```diff
@@ -778,10 +778,11 @@ public static boolean isUUID(String value)
 
 	/**
 	 * Is running from Eclipse
-	 * @return true if there is a directory org.adempiere.base within AdempiereHome (is the case when executed from Eclipse) 
+	 * @return true if there is a directory org.adempiere.base within AdempiereHome or if there is a System property org.idempiere.developermode set to Y 
 	 */
 	public static boolean isDeveloperMode() {
-		return Files.isDirectory(Paths.get(Adempiere.getAdempiereHome() + File.separator + "org.adempiere.base"));
+		return Files.isDirectory(Paths.get(Adempiere.getAdempiereHome() + File.separator + "org.adempiere.base"))
+				|| (!Util.isEmpty(System.getProperty("org.idempiere.developermode")) && System.getProperty("org.idempiere.developermode").equals("Y"));

```

Comment:
this can be simplified to `|| "Y".equals(System.getProperty("org.idempiere.developermode"))`

Reply:
Indeed, is better :)